src/main/resources/database/data.sql

```diff
@@ -40,6 +40,7 @@ INSERT INTO `user_role` (u_id, r_id) VALUES (2,5);
 INSERT INTO `employee`(u_id, p_id, e_forename, e_surname, e_patronymic, e_phone_number, e_info) VALUES (1,1,'Vasyl', 'Vasyliv', 'Vasylovich', '4509654345', 'blablabla1');
 INSERT INTO `employee`(u_id, p_id, e_forename, e_surname, e_patronymic, e_phone_number, e_info) VALUES (2,2,'Petro', 'Petriv',  'Petrovych',  '094586403', 'blablabla2');
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([plsql:S1192](https://sonarcloud.io/organizations/final-project/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=107&issues=AWW_98FYlgbYnohYhpGs&open=AWW_98FYlgbYnohYhpGs)

[ISSUE_KEY]: AWW_98FYlgbYnohYhpGs